### PR TITLE
Added backgrounds plus some small things too

### PR DESCRIPTION
**Pull request by AnAlgorithm**
Take a look at these. I may have goofed up in the code at some point of time but they look pretty okay. Do take a look at this and let me know if they're good.

### CODE REVIEW

**Comment by Looky1173** [comment]
I really like the new backgrounds! Did you make them yourself? 🙂

**Comment by AnAlgorithm** [comment]
> I really like the new backgrounds! Did you make them yourself? 🙂

@Looky1173 well, I just backpacked some scratch block art from one of ScratchCat's projects and then just changed the colour. I'm glad you liked it!

**Comment by Looky1173** [comment]
However...
image

**Comment by AnAlgorithm** [comment]
Like I said, I must have goofed up in the code somewhere. Or is it that SVG isn't supported in backgrounds?

File: assets/css/global.css [hunk 1]
```diff
@@ -48,7 +48,7 @@
     */
     --text: var(--polar-night-dark);
     --text-muted: var(--polar-night-very-light);
-    --background: var(--snow-storm-light);
+    --background: url('/bg/light.svg');
```
Comment (Looky1173) [inline assets/css/global.css:51]: Wrong path, the css folder is missing!

File: assets/css/global.css [hunk 2]
```diff
@@ -111,7 +111,7 @@
     */
     --text: var(--snow-storm-light);
     --text-muted: var(--snow-storm-dark);
-    --background: var(--polar-night-very-dark);
+    --background: url('/bg/dark.svg');
```
Comment (Looky1173) [inline assets/css/global.css:114]: Wrong path, the css folder is missing!

**Review by Looky1173 — CHANGES_REQUESTED** [review]

File: assets/css/global.css (hunk 1 above)
Comment (AnAlgorithm) [reply assets/css/global.css:51]: oh... you are supposed to write the name of the folder in which this code is contained? that explains all my bugs with other programs...

Comment (Looky1173) [reply assets/css/global.css:51]: Here's an example:
image

File: assets/css/global.css [hunk 3]
```diff
@@ -48,7 +48,7 @@
     */
     --text: var(--polar-night-dark);
     --text-muted: var(--polar-night-very-light);
-    --background: url('/bg/light.svg');
+    --background: url('css/bg/light.svg');
```
Comment (Looky1173) [inline assets/css/global.css:51]: Sorry, that still won't work. Here's how it's done:
image

File: assets/css/global.css [hunk 4]
```diff
@@ -111,7 +111,7 @@
     */
     --text: var(--snow-storm-light);
     --text-muted: var(--snow-storm-dark);
-    --background: url('/bg/dark.svg');
+    --background: url('css/bg/dark.svg');
```
Comment (Looky1173) [inline assets/css/global.css:114]: Same issue as above.

**Review by Looky1173 — CHANGES_REQUESTED** [review]

File: assets/css/global.css (hunk 1 above)
Comment (AnAlgorithm) [reply assets/css/global.css]: oh. also i have to put the ~ mark?
and if the css file (referring to the image) is in the root folder, what would you use?

File: assets/css/global.css [hunk 5]
```diff
@@ -48,7 +48,7 @@
     */
     --text: var(--polar-night-dark);
     --text-muted: var(--polar-night-very-light);
-    --background: url('css/bg/light.svg');
+    --background: url('~css/bg/light.svg');
```
Comment (Looky1173) [inline assets/css/global.css:51]: Sorry but you **must** include the assets folder so the correct path would be `~assets/css/bg/light.svg`.

File: assets/css/global.css [hunk 6]
```diff
@@ -111,7 +111,7 @@
     */
     --text: var(--snow-storm-light);
     --text-muted: var(--snow-storm-dark);
-    --background: url('css/bg/dark.svg');
+    --background: url('~css/bg/dark.svg');
```
Comment (Looky1173) [inline assets/css/global.css:114]: Same issue as above.

**Review by Looky1173 — CHANGES_REQUESTED** [review]

File: assets/css/global.css (hunk 5 above)
Comment (AnAlgorithm) [reply assets/css/global.css:51]: ok

**Review by Looky1173 — APPROVED** [review]

**Comment by Looky1173** [comment]
The issues are still the same. I think you've overlooked [where `var(--background)` is used](https://github.com/Looky1173/Itinerary/search?q=var%28--background%29).

**Comment by AnAlgorithm** [comment]
> The issues are still the same. I think you've overlooked [where `var(--background)` is used](https://github.com/Looky1173/Itinerary/search?q=var%28--background%29).

Found it.

**Comment by Looky1173** [comment]
Also, the dark mode SVG doesn't correspond with the previous dark mode background so it stands out too much. Please change the colours accordingly.
image
image

**Comment by AnAlgorithm** [comment]
@Looky1173 Done, ig

**Comment by Looky1173** [comment]
Contrast is still very low:
image
The colour of the original background is #242933 while the dark SVG's background colour is much lighter (#2e3240). Could you please change it to use the colour #242933?

**Comment by Looky1173** [comment]
Could you please do this for the light SVG's background colour as well (#eceff4 instead of #f9f9f9)?

**Comment by AnAlgorithm** [comment]
well, i fixed the dark one but something is preventing me from changing the light one. it's treating the bg as an image instead of an svg fill. I'll try fixing it though.

**Comment by Looky1173** [comment]
Let me do this for you! Here is the correct light themed SVG:
light-themed-svg

**Comment by Looky1173** [comment]
Also, I've optimised the size of the dark themed SVG:
dark-themed-svg

**Comment by AnAlgorithm** [comment]
cool! so is this finalized (and can be merged)?
also, did you take a look at the other changes?

**Comment by Looky1173** [comment]
> cool! so is this finalized (and can be merged)? also, did you take a look at the other changes?

Yes I did look at the other changes, they're good! Let me authorize Vercel to deploy a preview version of this, then we can see if everything is alright.

**Comment by Looky1173** [comment]
I personally like the changes; let me just ask in the Scratch forums for others' opinions. Then I will merge this. Also, the default random watercolour wallpapers could be changed to these to SVG backgrounds to make the UX more consistent and reduce the size of the website.

**Comment by AnAlgorithm** [comment]
as for the repeat of background, i can still do it.

**Comment by AnAlgorithm** [comment]
Also, I've just made sure that this block bg doesn't appear in the flame theme (because if it did it wouldn't go with the theme). I can make the background repeat, if you want.

**Comment by Looky1173** [comment]
> @Looky1173 Also, I've just made sure that this block bg doesn't appear in the flame theme (because if it did it wouldn't go with the theme). I can make the background repeat, if you want.

@AnAlgorithm Thank you; I think it looks good!

**Pull request merged**